src/libpriv/rpmostree-postprocess.c

```diff
@@ -192,17 +193,50 @@ find_kernel_and_initramfs_in_bootdir (GFile       *bootdir,
         }
     }
 
-  if (!ret_kernel)
+  ret = TRUE;
+  gs_transfer_out_value (out_kernel, &ret_kernel);
+  gs_transfer_out_value (out_initramfs, &ret_initramfs);
+ out:
+  return ret;
+}
+
+/* Given a directory @d, find the first child that is a directory,
+ * returning it in @out_subdir. */
+static gboolean
+find_first_subdirectory (GFile         *d,
+                         GFile        **out_subdir,
+                         GCancellable  *cancellable,
+                         GError       **error)
+{
+  gboolean ret = FALSE;
+  gs_unref_object GFileEnumerator *direnum = NULL;
+  gs_unref_object GFile *ret_subdir = NULL;
```

Comment:
We don't have a new enough libglnx submodule...I'd prefer to do that kind of thing at some point later as a bigger step, sound OK?